pkg/certificates/config.go

```diff
@@ -30,7 +30,7 @@ type CertClientConfig struct {
 
 // CertServerConfig is a config struct for use with a certificate signing service server
 type CertServerConfig struct {
-	Overwrite          bool   `default:"false" help:"if true, overwrites config AND authorization db is truncated"`
+	Overwrite          bool   `default:"false" help:"if true, overwrites config AND authorization db is truncated" setup:"true"`
```

Comment:
Do we still want to support overwrite here?

Reply:
^